### DISCUSSION TITLE
langsmith not working with AzureOpenAI in Playground

### DISCUSSION THREAD

**Discussion by cvgoudar** [langchain-ai/langsmith-sdk#238] [Q&A]
I am getting error while trying to use the Playground to improve the prompt. Playground is working with OpenAI API but not AzureOpenAI

Error: Invalid namespace: $ -> {"id":["langchain","chat_models","azure_openai","AzureChatOpenAI"],"lc":1,"type":"constructor","kwargs":{"model_name":"gpt-4","temperature":0,"model_version":"0613","openai_api_key":{"id":["OPENAI_API_KEY"],"lc":1,"type":"secret"},"deployment_name":"gpt-4","openai_api_base":"https://openaitest.openai.azure.com","openai_api_type":"azure","openai_api_version":"2023-07-01-preview"}}

I have used the AzureOpenAI key in "Secrets & API Keys" section

![image](https://github.com/langchain-ai/langsmith-sdk/assets/13920855/455583ff-a957-422c-aa6d-7184df87746f)

**Comment by hinthornw** [comment]
Going to mark this as stale! A lot has happened since then :)
